src/components/CreatePostModal/index.tsx

```diff
@@ -138,9 +143,8 @@ const CreatePostModal = ({ visible, onClose, userId, onSelect }: IModal) => {
     );
   };
 
-
   const handleEndReached = () => {
-    console.log('handleEndReached:')
+    console.log('handleEndReached:');
```

Comment:
you might be forgot to remove this line 😉

Reply:
Sure xD